src/banking/Server.kt

```diff
@@ -0,0 +1,168 @@
+                            }
+                        }
+                        AuthOperationsCode.ACCESS -> {
+                            resultMessage = try {
+                                val loginData = Auth.login(clientRequest.accountId ?: "", clientRequest.accountPassword)
+                                ServerMessage.LoginMessage(ServerMessageResultCode.SUCCESS, "You are now logged in!", loginData)
+                            } catch (e: Exception) {
+                                ServerMessage.LoginMessage(ServerMessageResultCode.ERROR, "Error logging into your account: " + (e.message ?: "Unknown error."), null)
+                            }
+                        }
+                    }
+                is ClientMessage.BankingMessage -> {
+                    try {
+                        val accountInformation = Auth.validate(clientRequest.jwtToken ?: "")
+                        if (accountInformation.id != clientRequest.originAccount)
+                            throw RuntimeException("The origin account id must be the same as the authenticated account id")
+                        var message = "Success!"
+                        when (clientRequest.operationCode) {
+                            BankingOperationsCode.DEPOSIT -> {
+                                deposit(clientRequest.originAccount, clientRequest.operationValue)
+                                message = "Successfully deposited the amount."
+                            }
+                            BankingOperationsCode.PIX -> {
+                                pix(clientRequest.originAccount, clientRequest.targetAccount ?: "", clientRequest.operationValue)
+                                message = "Successfully transfered the amount via pix."
+                            }
+                            BankingOperationsCode.TRANSFER -> {
+                                transfer(clientRequest.originAccount, clientRequest.targetAccount ?: "", clientRequest.operationValue)
+                                message = "Successfully transfered the amount."
+                            }
+                            BankingOperationsCode.WITHDRAW -> {
+                                withdraw(clientRequest.originAccount, clientRequest.operationValue)
+                                message = "Successfully withdrawed the amount."
+                            }
+                        }
+                        val account: Account = ServerState.internalState.accounts.first{ it.id == clientRequest.originAccount }
+                        resultMessage = ServerMessage.BankingMessage(ServerMessageResultCode.SUCCESS, message, account.value)
+                    } catch (e: Exception) {
+                        resultMessage = ServerMessage.BankingMessage(ServerMessageResultCode.ERROR, "Error realizing the operation: " + (e.message ?: "Unknown error."), null)
```

Comment:
"Error realizing the operation" -> "Error executing the operation".

Reply:
@AndreLaranjeira Você pode sugerir edição diretamente pelo Github, especialmente quando é algo assim

Reply:
Ok, vou tentar.